splines/gpuSplineUtils.cu

```diff
@@ -13,8 +13,8 @@
 #include <cuda_runtime.h>
 
 #ifdef CUDA_ERROR_CHECK
-#include <helper_functions.h>
-#include <helper_cuda.h>
+#include <Common/helper_functions.h>
+#include <Common/helper_cuda.h>
```

Comment:
What about this?
https://github.com/mach3-software/MaCh3/blob/212bc3d419015b54c8684b98ceb2b439913dbc4f/mcmc/gpuMCMCProcessorUtils.cu#L15

Reply:
Yep can fix that